cartography/intel/aws/redshift.py

```diff
@@ -97,6 +99,40 @@ def _attach_ec2_security_groups(neo4j_session, cluster, aws_update_tag):
         )
 
 
+def _attach_iam_roles(neo4j_session, cluster, aws_update_tag):
+    attach_cluster_to_role = """
+    MATCH (c:RedshiftCluster{id:{ClusterArn}})
+    MERGE (p:AWSPrincipal{arn:{RoleArn}})
+    MERGE (c)-[s:STS_ASSUMEROLE_ALLOW]->(p)
+    ON CREATE SET s.firstseen = timestamp()
+    SET s.lastupdated = {aws_update_tag}
+    """
+    for role in cluster.get('IamRoles', []):
+        neo4j_session.run(
+            attach_cluster_to_role,
+            ClusterArn=cluster['arn'],
+            RoleArn=role['IamRoleArn'],
+            aws_update_tag=aws_update_tag,
+        )
+
+
+def _attach_aws_vpc(neo4j_session, cluster, aws_update_tag):
+    attach_cluster_to_vpc = """
+    MATCH (c:RedshiftCluster{id:{ClusterArn}})
+    MERGE (v:AWSVpc{id:{VpcId}})
+    MERGE (c)-[m:MEMBER_OF_AWS_VPC]->(v)
+    ON CREATE SET m.firstseen = timestamp()
+    SET m.lastupdated = {aws_update_tag}
+    """
+    if 'VpcId' in cluster and cluster.get('VpcId'):
```

Comment:
this can be written as `if cluster.get('VpcId')`.  If `cluster['VpcId']` can't be None when it exists, `if 'VpcId' in cluster` is good enough